app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -403,6 +441,25 @@ public void newSurvey(View view) {
     }
 
     /**
+     * Called when the user clicks the exit Review button
+     */
+    public void exitReview(View view) {
+        ScoreRegister.clear();
+        beforeExit();
+        closeReviewFragment();
+    }
+
+    /**
+     * Checks if a survey fragment is active
+     */
+    private boolean isReviewFragmentActive() {
```

Comment:
@idelcano Can we merge isReviewFragmentActive and isSurveyFragmentActive into a single function and pass a parameter? Also can we simplify four lines inside the method into something like `return (currentFragment instanceof [Class])?`